src/assemble_workflow/assemble_args.py

```diff
@@ -31,9 +31,17 @@ def __init__(self):
             const=logging.DEBUG,
             dest="logging_level",
         )
+        parser.add_argument(
+            "-d",
+            "--distribution",
+            dest="distribution",
+            help="Choose distribution",
+            default="tar",
+        )
```

Comment:
Add `choices=self.SUPPORTED_DISTRIBUTIONS` and define  `SUPPORTED_DISTRIBUTIONS` similar to https://github.com/opensearch-project/opensearch-build/blob/c31d6a04ddcdbdb66a2f7ba9bfabdcf8c52f6d51/src/build_workflow/build_args.py#L13